### PR TITLE
README: Add notes on configuring for EU/UK versus US

### PR DESCRIPTION
**Pull request by ajlennon**
And comment on failure to OTA update being potentially related to ESP-IDF 5.x

Signed-off-by: Alex J Lennon <ajlennon@dynamicdevices.co.uk>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ Then open the project with VSCode.
 ### Configuration of the application
 
 The example application should first be configured to set at least:
-- `MENDER_SERVER_TENANT_TOKEN` to set the Tenant Token of your account on "https://hosted.mender.io" server;
+- `CONFIG_MENDER_SERVER_HOST` to set the server host. Default is valid for US `https://hosted.mender.io` and should be changed to `https://eu.hosted.mender.io` for EU/UK
```
Comment (joelguittet) [inline README.md:29]: Using the mender US server from Europe is working. The goal of this example is to be very simple and it's not a generic mender tutorial. Mender documentation is there for this purpose.

File: README.md [hunk 2]
```diff
@@ -52,7 +53,9 @@ E (41910) mender: ../components/mender-mcu-client/mender-mcu-client/src/mender-a
 I (41910) main: Mender client authentication failed (1/3)
 ```
 
-Which means you now have generated authentication keys on the device. Authentication keys are stored in NVS partition of the ESP32. You now have to accept your device on the mender interface. Once it is accepted on the mender interface the following will be displayed:
+Which means you now have generated authentication keys on the device. Authentication keys are stored in NVS partition of the ESP32. You now have to accept your device on the mender interface. If you are not seeing anything on your Mender dashboard make sure you are in the correct domain (US/EU) and have the correct `MENDER_SERVER_HOST` set.
```
Comment (joelguittet) [inline README.md:56]: See above comment.

File: README.md [hunk 3]
```diff
@@ -122,6 +125,8 @@ I (14811) mender: ../components/mender-client/mender-client/src/mender-client.c
 
 Congratulation! You have updated the device. Mender server displays the success of the deployment.
 
+If the update is downloaded, verified, applied, and on reset you are still running the old version this might be because you have build with ESP-IDF 5.x SDK which as indicated above is not supported. You need to build with ESP-IDF 4.x SDK
```
Comment (joelguittet) [inline README.md:128]: I'm not sure why adding this. Actually this even d'ont build at all with ESP-IDF 5.x on my side. Can you detail your experience which lead you writing this ? Thanks a lot !

Comment (ajlennon) [reply README.md:128]: No worries buddy - I'll make my own example for the rest of the world then.

File: README.md (hunk 1 above)
Comment (ajlennon) [reply README.md:29]: Easy enough - I'll do my own example for the non-American parts of the world 🤣 Laterz

Comment (joelguittet) [reply README.md:29]: Not sure of the goal of your answer, but you can do what you want in your own fork. Just for your information I'm French and living in France ;-)

Comment (ajlennon) [reply README.md:29]: Yeah I just want something that works :) Will do

File: README.md (hunk 3 above)
Comment (joelguittet) [reply README.md:128]: Additional information: since mender-mcu-client 0.2.0 ESP-IDF v5.0.x are supported.

**Pull request closed without merging**